src/control_filters/rate_limiter_parameters.yaml

```diff
@@ -3,38 +3,70 @@ rate_limiter:
     type: double,
     description: "Sampling interval in seconds",
     validation: {
-      gt<>: [0.0]
+      gt<>: [0.0],
     },
   }
   max_value: {
     type: double,
     default_value: .NAN,
+    description: "Maximum value, e.g. [m/s]",
+    validation: {
+      "control_filters::gt_eq_or_nan<>": [0.0]
+    },
```

Comment:
in principle you are right, and it would already work setting it + or -infinite. However, NaN is the default value of the diff_drive_controller. And the logic of "copying the positive value if negative is not given" would not work because infinite is a valid limit 🤔

Reply:
Let's leave it as it is then